.github/workflows/pull-request.yml

```diff
@@ -84,9 +109,26 @@ jobs:
         key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
         restore-keys: ${{ runner.os }}-gradle
     - name: Build with Maven
-      run: ./mvnw -B install javadoc:javadoc-no-fork --file pom.xml -Pcode-coverage,jdk8-tests -Dtest.log.level=WARN
       env:
         SPARK_LOCAL_IP: localhost
+      run: |
```

Comment:
I just realised what jdk8 tests does. Isn't this the only way that hive tests are run? I think we have to keep this in CI. And anyways, I thikn we should remove `hmsbridge` modules

Reply:
I am of the same opinion as @rymurr : if someone modify the hive code (or any code the hive code depends on), we should not force people to figure out which tag to add to have an accurate CI picture